packages/definitions-parser/src/get-affected-packages.ts

```diff
@@ -29,7 +29,11 @@ export async function getAffectedPackages(
     return { errors };
   }
   const { additions, deletions } = git;
-  const addedPackageDirectories = mapDefined(additions, (id) => id.typesDirectoryName);
+  const addedPackageDirectories = mapDefined(additions, (pkg) => {
+    if (!pkg.typesDirectoryName) return undefined;
+    if (!pkg.version || pkg.version === "*") return pkg.typesDirectoryName;
+    return `${pkg.typesDirectoryName}/v${formatTypingVersion(pkg.version)}`;
+  });
```

Comment:
```suggestion
  const addedPackageDirectories = mapDefined(additions, (pkg) => pkg.subDirectoryPath);
```

Reply:
Ah. I guess you could move the implementation of `get subDirectoryName()` to a utility.

Reply:
Yeah though the signals there are different because the real package uses a Boolean for "latest" instead of the star... The whole thing could use some TLC